ro_help/hub/templates/partials/social_sharing.html

```diff
@@ -0,0 +1,13 @@
+{% load social_share %}
+<div id="social-sharing">
+    Share on:
```

Comment:
missing translation tag

Reply:
I've skipped this on purpose. Do we want to translate "Share on"? Seems to me that this is the kind of string that's recognized in any language without actually translating it. 

Something similar to _Like_ string which would although recognizable as _Apreciaza_ makes more sense in it's english version.